src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -72,6 +72,10 @@
     @Argument(fullName = StandardArgumentDefinitions.DISABLE_SEQUENCE_DICT_VALIDATION_NAME, shortName = StandardArgumentDefinitions.DISABLE_SEQUENCE_DICT_VALIDATION_NAME, doc = "If specified, do not check the sequence dictionaries from our inputs for compatibility. Use at your own risk!", optional = true, common = true)
     private boolean disableSequenceDictionaryValidation = false;
 
+    public static final String ERROR_ON_OUT_OF_DATE_INDEX = "errorOnOutOfDateIndex";
+    @Argument(fullName = ERROR_ON_OUT_OF_DATE_INDEX, shortName = ERROR_ON_OUT_OF_DATE_INDEX, doc = "Throw an error when the index file is detected to be out of date.", optional = true, common = true)
```

Comment:
Perhaps amend this comment to clarify that out of date means older than the indexed file.